README.md

```diff
@@ -56,24 +56,24 @@ in the docs for examples and integrations.
 
 ## What we are about
 
-We have several primary objectives with this linter:
+We have several primary objectives with regards to this linter:
```

Comment:
That's not about this linter. But that's about the code that will be produces with the help of this linter.

Reply:
So, it might be just `We have several primary objectives`

Reply:
Yeah, I was not sure in regards to what you have those objectives. Cool, will fix it :)